app/themes/components.tsx

```diff
@@ -154,16 +168,10 @@ export const components: Components = {
         display: "flex",
         gap: 12,
         margin: 0,
-        padding: 12,
-        paddingLeft: 16,
-        transition: "background-color 0.2s ease",
-
-        "&:not(.Mui-disabled):hover": {
-          backgroundColor: palette.cobalt[50],
-        },
       },
+      // @ts-ignore
```

Comment:
Do you know why we need ts-ignore here when we did not need it before ?

Reply:
Good catch, it was not needed anymore here, I'll remove it in next commits – there's a type error when spreading non-native typography variant styles in style overrides, I think before there was a `...typography.body3` here (or I had it locally for a while, before extracting it to `useStyles`) 🤔